compact/far_backward/magnets.xml

```diff
@@ -35,62 +35,99 @@
+      </magnet>
+      <magnet id="2" name="Magnet_B3eR"
+        x="B3eR_CenterX" y="0" z="B3eR_CenterZ" theta="B3eR_Theta"
+        length="B3eR_Length" rin="B3eR_InnerRadius" rout="B3eR_OuterRadius">
+      </magnet>
+      <magnet id="3" name="Magnet_B4eR"
+        x="B4eR_CenterX" y="0" z="B4eR_CenterZ" theta="B4eR_Theta"
+        length="B4eR_Length" rin="B4eR_InnerRadius" rout="B4eR_OuterRadius">
+      </magnet>
+      <magnet id="4" name="Magnet_B5eR"
+        x="B5eR_CenterX" y="0" z="B5eR_CenterZ" theta="B5eR_Theta"
+        length="B5eR_Length" rin="B5eR_InnerRadius" rout="B5eR_OuterRadius">
+      </magnet>
+      <magnet id="5" name="Magnet_B6eR"
+        x="B6eR_CenterX" y="0" z="B6eR_CenterZ" theta="B6eR_Theta"
+        length="B6eR_Length" rin="B6eR_InnerRadius" rout="B6eR_OuterRadius">
+      </magnet>
+      <magnet id="6" name="Magnet_B7eR"
+        x="B7eR_CenterX" y="0" z="B7eR_CenterZ" theta="B7eR_Theta"
+        length="B7eR_Length" rin="B7eR_InnerRadius" rout="B7eR_OuterRadius">
+      </magnet>
     </detector>
 
 
-    <comment> Inner vacuum for magnets </comment>
-
-    <detector id="Pipe_to_Q1eR_ID" name="Pipe_to_Q1eR" type="BackwardsBeamPipe">
-      <Pipe wall_thickness="2*mm" outerD1="Q1eR_InnerRadius*2" outerD2="Center_Beampipe_Rad*2"
-        end1z="Center_Beampipe_End" end2z="Q1eR_CenterPosition+Q1eR_Length/2"/>
-    </detector>
 
-    <detector id="Pipe_in_Q1eR_ID" name="Pipe_in_Q1eR" type="BackwardsBeamPipe">
-      <Pipe wall_thickness="2*mm" outerD1="Q1eR_InnerRadius*2" outerD2="Q1eR_InnerRadius*2"
-        end1z="Q1eR_CenterPosition+Q1eR_Length/2" end2z="Q1eR_CenterPosition-Q1eR_Length/2"/>
+    <comment> Hadron side beam magnet volumes </comment>
+    <!--
+    <detector name="Magnet_Q1APR" type="ip6_CylindricalDipoleMagnet" vis="RedVis">
+    <placement  x="(Q1APR_StartX+Q1APR_EndX)/2" y="0" z="(Q1APR_StartZ+Q1APR_EndZ)/2" theta="Q1APR_Theta"/>
+    <dimensions x="Q1APR_InnerRadius*4" y="Q1APR_InnerRadius*4" z="Q1APR_Length" r="2.0*Q1APR_InnerRadius"/>
+    <apperture  x="Q1APR_InnerRadius*2" y="Q1APR_InnerRadius*2" r="Q1APR_InnerRadius"/>
+    <coil dx="2*cm" dy="1.5*cm" />
     </detector>
-
-    <detector id="Pipe_Q1eR_to_Q2eR_ID" name="Pipe_Q1eR_to_Q2eR" type="BackwardsBeamPipe">
-      <Pipe wall_thickness="2*mm" outerD1="Q2eR_InnerRadius*2" outerD2="Q1eR_InnerRadius*2"
-        end1z="Q1eR_CenterPosition-Q1eR_Length/2" end2z="Q2eR_CenterPosition+Q2eR_Length/2"/>
+    -->
+
+    <!--
+    <detector name="Magnet_Q1BPR" type="ip6_CylindricalDipoleMagnet" vis="RedVis">
+    <placement  x="(Q1BPR_StartX+Q1BPR_EndX)/2" y="0" z="(Q1BPR_StartZ+Q1BPR_EndZ)/2" theta="Q1APR_Theta"/>
+    <dimensions x="Q1BPR_InnerRadius*4" y="Q1BPR_InnerRadius*4" z="Q1BPR_Length" r="2.0*Q1BPR_InnerRadius"/>
+    <apperture  x="Q1BPR_InnerRadius*2" y="Q1BPR_InnerRadius*2" r="Q1BPR_InnerRadius"/>
+    <coil dx="2*cm" dy="1.5*cm" />
     </detector>
+    -->
```

Comment:
What is the purpose of this commented code? Can it be removed?

Reply:
It was commented out previously by someone else.  Doesn't really need to be there and can be easily added back in later (Q1APR and Q1BPR magnets).

Reply:
For information - These are commented out as they overlap with the electron beamline magnets with the token outer radii they have been given. This will probably remain the case until we get the full CAD drawings including the cryogenics and can import/take measurements from there.